HouseRules_Core/HangoutsGameRacer.cs

```diff
@@ -185,7 +185,7 @@ private static void PrepareToLeaveHangouts(GroupLaunchTable groupLaunchTable)
 
             // Recreating `GameStateHobbyShop.ExitBowser`
             var teleport = Traverse.Create(_gameStateHobbyShop).Field<Teleport>("teleport").Value;
-            teleport.SetTeleportationEnabled(enabled: false);
+            teleport.UpdateState(true, false);
```

Comment:
nit: for clarity on what booleans do, could we inline parameter names?  (e.g., ...UpdateState(paramName1: true, paramName2: false) (don't have access to the code ATM so don't know what these are :p).